Verification and identity endpoints

brettmortensen opened the pull request:
Adds 5 endpoints for verification and identity. (verify, account numbers by account guid, account numbers by member guid, identify, and account owners)

@film42

brettmortensen committed:
Verification and identity endpoints

Adds 4 endpoints for verification and identity. (verify,
account_numbers, identify, and account_owners)

film42 commented on client/account_number.go:
```diff
@@ -0,0 +1,42 @@
+
+	if response.StatusCode == 200 {
+		accountNumbersResponse := &models.AccountNumbersResponse{}
+		json.Unmarshal([]byte(bufferStr), accountNumbersResponse)
```
`json.Unmarshal` returns an `error`, so you're going to need to check that and return if not `nil`. Same for a few other places in this PR.

film42 replied:
Otherwise, LGTM

brettmortensen replied:
Good catch. Error handling has been added.

brettmortensen committed:
Account numbers by account endpoint, error handling

Adds account numbers by account guid endpoint which is part of account
verification. Also adds error handling to `json.Unmarshal` references.

brettmortensen committed:
Consolidate identical functions

Moves identical `parseAccountNumbersResponse` function to the `account_number`
model so it can still be referenced in the `client/account.go` and
`client/member.go` files.

brettmortensen committed:
Account Numbers endpoint by account or member guid

Added logic into the `ListAccountNumbers` endpoint to check if it was
passed a account or member GUID, as we have endpoints that handle both.

brettmortensen committed:
Remove duplicate function

film42 reviewed (approved):
Looks good!

brettmortensen committed:
Refactor account numbers/owners

Removes account number conditional in order to have 2 account number
endpoints (by account guid and by member guid) that follows the setup of
our API. Rename account owners endpoint from `listAccountOwners` to
`listMemberAccountOwners` to follow existing naming pattern.
